fastly/ngwaf/v1/workspaces/api_response.go

```diff
@@ -7,10 +7,10 @@ import "time"
 type Workspace struct {
 	// AttackSignalThresholds are the parameters for system site
 	// alerts.
-	AttackSignalThresholds AttackSignalThresholds `json:"attack_signal_thresholds"`
+	AttackSignalThresholds *AttackSignalThresholds `json:"attack_signal_thresholds,omitempty"`
 	// ClientIPHeaders lists the request headers containing the
 	// client IP address.
-	ClientIPHeaders []string `json:"client_ip_headers"`
+	ClientIPHeaders *[]string `json:"client_ip_headers,omitempty"`
```

Comment:
Slices are already pointers, so there's no need to make this a pointer. If there are no headers in the response, the slice will be empty.